app/views/templates/rulingCertificateTemplate.scala.html

```diff
@@ -20,33 +20,28 @@
 @(appConfig: FrontendAppConfig, c: Case, d: Decision, getCountryName: String => Option[String],
 compositeMode: Boolean = false)(implicit messages: Messages)
 
-@createLink(path: String)(implicit req: play.api.mvc.Request[_]) = @{
-val protocol = if(req.secure) "https" else "http"
-s"$protocol://${req.host}${routes.Assets.versioned(path)}"
-}
-
-<div id="print-pages" class="print-pages">
-    <div id="ruling-certificate" class="print-document" id="print-document">
+<div class="print-pages">
+    <div class="print-document" id="print-document">
         <div class="govuk-grid-row print-header">
             <div class="govuk-grid-column-one-third">
                 <img class="crown-logo-atar" alt="HM Revenue and Customs" src="@messages("logo.encoded")"/>
             </div>
             <div>
-                <h2 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
+                <h1 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
                     @messages("pdf.ruling.certificate.title")
-                </h2>
+                </h1>
             </div>
         </div>
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full mt-1">
                 <span class="govuk-caption-xl">@messages("pdf.ruling.certificate.rulingFor", c.reference)</span>
-                <h1 class="govuk-heading-xl">@c.application.goodName</h1>
+                <p class="govuk-heading-xl">@c.application.goodName</p>
```

Comment:
please revert `<p>` back to `<h2>` and resize by changing class to `"govuk-heading-l"`

Reply:
used `govuk-heading-xl` to make it look like in staging